docs/source/60_building_apps.md

```diff
@@ -15,15 +16,18 @@ This document covers three patterns for building applications downstream of ADAM
 
 ADAM's CLI is implemented in the adam-cli Apache Maven module of the
 [bdgenomics/adam](https://github.com/bigdatagenomics/adam) repository, one
-.scala source file for each CLI action (e.g. [Transform.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala)
-for the [transform](#transform) action), and a main class ([ADAMMain.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala))
+.scala source file for each CLI action (e.g.
+[TransformAlignments.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala)
+for the [transform](#transform) action), and a main class
+([ADAMMain.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala))
 that assembles and delegates to the various CLI actions.
 
 To add a new command:
 
-Extend `Args4jBase` class to specify arguments to the command. Arguments are defined using
-the [args4j library](http://args4j.kohsuke.org/). If reading from or writing to Parquet,
-consider including Parquet arguments via `with ParquetArgs`.
+Extend `Args4jBase` class to specify arguments to the command. Arguments are
```

Comment:
Alternatively, numbered list under `To add a new command:` and add subsection headings? e.g.,

```
To add a new command:

1. [Extend Args4jBase to specify arguments](#extend-arguments)
2. [Extend BDGCommandCompanion](#extend-companion)

...

####  Extend Args4jBase to specify arguments {#extend-arguments}
```